open-company/open-company-web
# Revisions cache and animation

card: https://trello.com/c/oSxluVR8

Test:
- try to change revisions on some sections (make sure to test finances, growth and one of the others)
- check on every revision if the data are editable (only normal sections, no notes sections for now)
- run the tests

Also this fix the first check on the Growth checklist here https://trello.com/c/hrUDr6WC since it was causing an error
